src/server/req_manager.c

```diff
@@ -3796,6 +3799,12 @@ struct batch_request *preq;
 			return;
 		}
 	}
+	is_node_ip = inet_pton(AF_INET, preq->rq_ind.rq_manager.rq_objname, &(check_ip.sin_addr));
+	if(is_node_ip > 0) {
+		log_eventf(PBSEVENT_DEBUG, PBS_EVENTCLASS_NODE, LOG_INFO, preq->rq_ind.rq_manager.rq_objname,
+			"Add PBS_MOM_NODE_NAME='IP address' in /etc/pbs.conf in Mom host and restart MOM to avoid node inconsistencies.");
```

Comment:
shall we say - "Node added using IP address\nVerify that PBS_MOM_NODE_NAME is configured on the respective host"